bitnami/vault/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: vault
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/vault
-version: 1.4.30
+version: 1.4.31
```

```suggestion
version: 1.5.0
```

Bumping version to a new minor as you are adding a new parameter